### ISSUE TITLE
Can’t halt, suspend, destroy, provision, etc.

### ISSUE THREAD

**Issue by allancole** [bradp/vv#260]
So, I’ve been using VVV for months without issue and yesterday, I installed VV following the instructions mentioned in your read me here. It seems to install correctly without error however, once I created a new site, it appears to break VVV. When I try to visit the newly created site in my browser, it shows this error: https://cloudup.com/cQOxi7H2mU8 

Then, when I try to visit the standard VVV site (local.wordpress.dev), it shows a “404 Error | nginx”. Before creating a new site with VV, this showed a working WordPress install.

Also, I can no longer use the halt, reload, destroy, provision, etc. commands. They all return an error as follows (replacing `'vagrant_halt'` with appropriate command): 

```
==> default: Running triggers before reload...
==> default: Executing command "vagrant ssh -c vagrant_halt"...
==> default: /home/vagrant/.bash_profile: line 39: /srv/www/wp-cli/utils/wp-completion.bash: No such file or directory
==> default: /home/vagrant/bin/db_backup: line 12: /srv/database/backups/wordpress_default.sql: No such file or directory
==> default: /home/vagrant/bin/db_backup: line 12: /srv/database/backups/wordpress_develop.sql: No such file or directory
==> default: /home/vagrant/bin/db_backup: line 12: /srv/database/backups/wordpress_trunk.sql: No such file or directory
==> default: Command execution finished.
The local command "vagrant ssh -c 'vagrant_halt'" returned a failed exit
code or an exception. The error output is shown below:

/home/vagrant/.bash_profile: line 39: /srv/www/wp-cli/utils/wp-completion.bash: No such file or directory
/home/vagrant/bin/db_backup: line 12: /srv/database/backups/wordpress_default.sql: No such file or directory
/home/vagrant/bin/db_backup: line 12: /srv/database/backups/wordpress_develop.sql: No such file or directory
/home/vagrant/bin/db_backup: line 12: /srv/database/backups/wordpress_trunk.sql: No such file or directory
```

Rebooting my computer is the only way to destroy the machine. But when I run `vagrant up` after a reboot, it kicks out a “Completed but with errors” message and the same command issues mentioned above persist. I can confirm all of the files that are mentioned in the error exist in my `/vagrant-local` directory, but it seems like the script is looking for them in the `/srv` which doesn’t seem right.

VV setup details

```
vv version 1.9.3

vv: /usr/local/bin/vv

vvv path: ~/Dropbox/Sites/vagrant-local/
home: /Users/allancole

tput: /usr/bin/tput
cat: /bin/cat
curl: /usr/bin/curl
brew: /usr/local/bin/brew
tar: /usr/bin/tar
find: /usr/bin/find
git: /usr/bin/git
sed: /usr/bin/sed
paste: /usr/bin/paste
vagrant: /usr/local/bin/vagrant
```

Appreciate your help.

**Comment by bradp** [comment]
Have you customized your VVV provisioning script at all? These all seem to be VVV-specific errors.

Can you also provide the full text you get when doing a `vagrant up --provision`. An easy way is `vagrant up --provision >> log.txt`. Thanks!

**Comment by allancole** [comment]
> Have you customized your VVV provisioning script at all? 

Nope. Sort of new to this so just trying to get the standard setup working.

Here’s results of the `vagrant up --provision` command

```
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: VirtualBox VM is already running.
==> default: [vagrant-hostsupdater] Checking for host entries
==> default: [vagrant-hostsupdater]   found entry for: 192.168.50.4 vvv
==> default: [vagrant-hostsupdater]   found entry for: 192.168.50.4 vvv.dev
==> default: [vagrant-hostsupdater]   found entry for: 192.168.50.4 local.wordpress.dev
==> default: [vagrant-hostsupdater]   found entry for: 192.168.50.4 local.wordpress-trunk.dev
==> default: [vagrant-hostsupdater]   found entry for: 192.168.50.4 src.wordpress-develop.dev
==> default: [vagrant-hostsupdater]   found entry for: 192.168.50.4 build.wordpress-develop.dev
==> default: [vagrant-hostsupdater]   found entry for: 192.168.50.4 woothemes.dev
==> default: Running triggers after up...
==> default: Executing command "vagrant ssh -c vagrant_up"...
==> default: /home/vagrant/.bash_profile: line 39: /srv/www/wp-cli/utils/wp-completion.bash: No such file or directory
==> default: Command execution finished.
```

One thing that has changed since you responded is I manually edited my hostfile to include an entry for `woothemes.dev` Seems to have fixed the missing database file errors but the wp-cli issue is still there. I still get the same 404 errors mentioned earlier but now the error shows this, when I run a command: 

> These all seem to be VVV-specific errors.

Yeah, thats what it looks like but none of these errors existed before installing VV. I can direct this to VVV support if you prefer. Let me know.

**Comment by allancole** [comment]
So I managed to sort it out with the help of a colleage of mine. 

I had my VVV folder placed inside of a Dropbox directory (with folder syncing turned off). I just started over from scratch with VVV + VV in a new directory outside of Dropbox and everything seems to be working better now. I am able to create new site instances and I no longer receive the issues mentioned above. 

Not sure if this is actually a bug or not, but I’m going to go ahead and close this since things are working now. 

Thanks for responding so quickly!
